internal/pkg/service/templates/dependencies/mocked.go

```diff
@@ -19,8 +27,20 @@ func NewMockedAPIScope(t *testing.T, cfg config.Config, opts ...dependencies.Moc
 	var err error
 	cfg.StorageAPIHost = mocked.StorageAPIHost()
 	cfg.PublicAddress, err = url.Parse("https://templates.keboola.local")
-	cfg.Etcd = mocked.TestEtcdCredentials()
 	require.NoError(t, err)
+	cfg.Etcd = mocked.TestEtcdCredentials()
+
+	// Prepare test repository with templates, instead of default repositories, to prevent loading of all production templates.
+	if reflect.DeepEqual(cfg.Repositories, config.DefaultRepositories()) {
+		tmpDir := t.TempDir()
+		assert.NoError(t, aferofs.CopyFs2Fs(nil, filesystem.Join("git_test", "repository"), nil, tmpDir))
+		assert.NoError(t, os.Rename(filepath.Join(tmpDir, ".gittest"), filepath.Join(tmpDir, ".git"))) // nolint:forbidigo
+		cfg.Repositories = []model.TemplateRepository{{
+			Type: model.RepositoryTypeGit, Name: "keboola", URL: fmt.Sprintf("file://%s", tmpDir), Ref: "main",
+		}}
+	}
```

Comment:
V CI mi niektore testy timeoutovali, lebo sa paralelne zacali klonovat a nacitavat vsetky produkcne `templates`, co v teste nechceme. Trva to niekolko `sec`, a v produkcii sa to deje iba pri starte servera a na pozadi kazdych `5 min`, ak doslo k zmene.

Testy bezia paralelne, ... ked sa aj toto klonovanie spustilo viac krat paralelne, tak to zablokovalo test na `10-tky sec`.